### ISSUE TITLE
training keyword using training_models.ipynb file epoch and batch size recommendation required.

### ISSUE THREAD

**Issue by BilalZubairi** [dscripka/openWakeWord#273]
### SUMMARY
I trained a custom wake word model for the keyword "hilfe" using the openWakeWord framework following the Alexa-style FCN model.I’m looking for guidance on how to reduce false positives and improve recall, particularly tuning hyperparameters like learning rate, layer size, and number of epochs for the training_models.ipynb on the following number of samples.

- Positive samples (containing 61653 audios)
- Negative samples (containing 290008 audios)

### Current Training Results
1. Final Accuracy: 0.9818
2. Final Recall: 0.9397
3. False Positives per Hour (FPH): 27.5888
### Parameters used
1. Epochs: 500
2. Layer size: 128
3. Learning rate : 0.001
### Dataset Information
1. Wake word: "hilfe"
2. Positive samples: 61,653
3. Negative samples: 290,008
4. Total training samples: (351,661)
5. Clip duration: 1 seconds
6. Mel shape: (3, 96)

**What hyperparameters should I tune (e.g., layer_dim, learning_rate, dropout, epochs, batch_size) to reduce false positives while maintaining the recall?**
